networkx/generators/lattice.py

```diff
@@ -145,25 +145,30 @@ def grid_graph(dim, periodic=False):
 
 @nx._dispatchable(graphs=None, returns_graph=True)
 def hypercube_graph(n):
-    """Returns the *n*-dimensional hypercube graph.
-
-    The nodes are the integers between 0 and ``2 ** n - 1``, inclusive.
+    """
+    Return the *n*-dimensional hypercube graph.
 
-    For more information on the hypercube graph, see the Wikipedia
-    article `Hypercube graph`_.
 
-    .. _Hypercube graph: https://en.wikipedia.org/wiki/Hypercube_graph
+    The n-dimensional hypercube graph has 2^n nodes, each represented as an n-tuple
```

Comment:
Should we italicize the n's here too for consistency? 

i.e.

```suggestion
    The *n*-dimensional hypercube graph has 2^n nodes, each represented as an *n*-tuple
```